ui/apps/platform/src/Containers/NetworkGraph/components/CIDRFormModalProvider.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { createContext, useContext, useState } from 'react';
+
+const defaultValue = {
+    isCIDRFormModalOpen: false,
+    initialCIDRFormValue: '',
+    toggleCIDRFormModal: () => {},
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    setInitialCIDRFormValue: (value: string) => {},
```

Comment:
I understand we don't plan to use this context without a provider, but typing it would probably be safer and we wouldn't need to disable eslint here.

```tsx
interface CIDRFormModalContextType {
    isCIDRFormModalOpen: boolean;
    initialCIDRFormValue: string;
    toggleCIDRFormModal: () => void;
    setInitialCIDRFormValue: Dispatch<SetStateAction<string>>;
}
const defaultValue = {
    isCIDRFormModalOpen: false,
    initialCIDRFormValue: '',
    toggleCIDRFormModal: () => {},
    setInitialCIDRFormValue: () => {},
};
const CIDRFormModalContext = createContext<CIDRFormModalContextType>(defaultValue);
```